src/components/Pages/Admin/View.vue

```diff
@@ -53,6 +97,78 @@
 </template>
 
 <script>
+import { mapGetters, mapActions } from 'vuex';
+import feathersClient from '../../../feathers-client';
+import GeogebraViews from '../../Geogebra/GeogebraViews';
+// import ToastrMixin from '@/mixins/ToastrMixin.vue';
+const api = feathersClient;
+
 export default {
+    // name: 'View',
+    data() {
+        return {
+            code: undefined,
+            groupsInClass: undefined,
+            GV: undefined,
+        };
+    },
+    // mixins: [ToastrMixin],
+    computed: {
+        ...mapGetters('classes', {
+            findClassesInStore: 'find',
+            getClass: 'get',
+        }),
+
+        ...mapGetters('groups', {
+            findGroupsInStore: 'find',
+        }),
+
+        ...mapGetters('users', {
+            user: 'current',
+        }),
+
+        classes() {
+            return this.findClassesInStore({
+                query: { $sort: { code: 1 } },
+            });
+        },
+    },
+    methods: {
+        ...mapActions('classes', {
+            findClasses: 'find',
+        }),
+
+        ...mapActions('groups', {
+            findGroups: 'find',
+        }),
+
+        async selectAllGroupsinClass(code) {
+            // this.clearToast();
+            this.code = code;
+            this.groupsInClass = await this.findGroups({ query: { class: code } });
+            if (!this.groupsInClass.length) {
+                // this.showToast('Selected class nas no groups', 'warning');
+            }
+
+            this.loadApplets();
+        },
+
+        async loadApplets() {
+            if (this.groupsInClass) {
```

Comment:
Those two `if` statements can be merged into one with `&&`. Additionally, it is enough to check that:

`if (array && array.length)` - `length` will be cast to `true` when larger than 0.